ieeetran/footer.tex

```diff
@@ -5,10 +5,27 @@
 \printbibliography
 \fi
 
+\appendices
```

Comment:
I can't find any reference for this command - shouldn't this be inside the appendix test below though?